setup.py

```diff
@@ -23,6 +23,7 @@
         'lmdb>=0.94,<1.0.0',
         'regex>=2017.9.23',
         'PyYAML>=3.12,<4.0',
+        'sphinx==1.7.0',
```

Comment:
we'll need to remove this once sphinx 1.7.6 is released and RTD is no longer using 1.7.4 by default.